examples/resources/awscc_config_config_rule/config_config_rule.tf

```diff
@@ -0,0 +1,37 @@
+resource "awscc_config_config_rule" "r" {
+  config_rule_name = ""
```

Comment:
empty string for resource name is not recommended, please add some meaningful resource name